### DISCUSSION TITLE
Finer Fan Controls

### DISCUSSION THREAD

**Discussion by Smokescreen5gh** [BartoszCichecki/LenovoLegionToolkit#914] [Ideas]
Is it possible to add more finer control for fan curves in terms of setting custom fan rpms.

I think the base fan speed is 1700 rpm. It would be nice if we can get the option to set a lower fan speed than that.

**Comment by BartoszCichecki** [comment]
This idea was already discussed here: https://github.com/BartoszCichecki/LenovoLegionToolkit/issues/696.
